### PR TITLE
[Docs] Japanese translation of docs/feature_thermal_printer.md

### PR DESCRIPTION
**Pull request by umi-umi**
## Description

Add a Japanese translation of docs/feature_thermal_printer.md.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by mtei** [comment]
@shelaf @s-show @hsgw @satt99 @takashiski @yoshihara よかったらレビューお願いします。

File: docs/ja/feature_thermal_printer.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  git diff 0.8.147 HEAD -- docs/feature_thermal_printer.md | cat
+-->
+
+<!-- FIXME: Describe thermal printers support here. -->
+
+## 感熱式プリンタのキーコード
+
+| キー | 説明 |
+|-----------|----------------------------------------|
+| `PRINT_ON` | ユーザが入力した全ての印刷を開始 |
```
Comment (s-show) [inline docs/ja/feature_thermal_printer.md:14]: 感熱式プリンタの処理を行なっていると思われる `quantum/process_keycode/process_printer_bb.c` のソースコードを見ると、どうもユーザーが入力したものは軒並み印刷するようですので、次のような訳はどうでしょうか。
```suggestion
| `PRINT_ON` | ユーザの入力を全て印刷開始 |
```

File: docs/ja/feature_thermal_printer.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+| `PRINT_OFF` | ユーザが入力した全ての印刷を停止 |
```
Comment (s-show) [inline docs/ja/feature_thermal_printer.md:15]:
```suggestion
| `PRINT_OFF` | ユーザの入力の印刷を停止 |
```

File: docs/ja/feature_thermal_printer.md (hunk 1 above)
Comment (umi-umi) [reply docs/ja/feature_thermal_printer.md:14]: ご指摘ありがとうございます。元の訳だとプリンタキュー的なものがあるようなイメージになってしまいますね。
訳例ありがとうございます。

**Comment by mtei** [comment]
Looks good to me.  I think this is ready to merge.

**Comment by noroadsleft** [comment]
ありがとうございました!

**Pull request merged**